docker/images/infra-task/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG base_image=ruby:2.7.3
+FROM ${base_image}
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential && apt-get clean
```

Comment:
```suggestion
# TODO: use the govuk-ruby base/builder images once they're ready, so that we
# can avoid bloating the image with package updates.
RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential && apt-get clean
```